### PR TITLE
pass json-compatible object to json_response

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?180877

@dannyroberts

### CODE REVIEW

File: corehq/apps/userreports/views.py [hunk 1]
```diff
@@ -385,7 +385,7 @@ def import_report(request, domain):
 def report_source_json(request, domain, report_id):
     config = get_document_or_404(ReportConfiguration, domain, report_id)
     config._doc.pop('_rev', None)
-    return json_response(config)
+    return json_response(config._doc)
```
Comment (dannyroberts) [inline corehq/apps/userreports/views.py]: This doesn't seem like the right fix to me. Something inside config._doc is not actually json serializable somehow, and I think we need to figure out what that is. Anything that's json serializable can be `deepcopy`d and it's essentially failing on `deepcopy(config._doc)`.

Comment (czue) [inline corehq/apps/userreports/views.py]: (it's a lazy translation - which `json_response` handles in the json encoder)

it does seem curious that a translation would end up in an `jsonobject` though.

Comment (nickpell) [inline corehq/apps/userreports/views.py]: @czue How do you know it's a translation?

Comment (czue) [inline corehq/apps/userreports/views.py]: i think maybe i'm wrong actually. i thought i had seen that error before but looking closer it seems like i might just be misremembering.

**Comment by nickpell** [comment]
looks similar to https://github.com/dimagi/jsonobject/issues/115

**Comment by dannyroberts** [comment]
Sort of. That jsonobject issue is about not being able to deepcopy doc.some_dict_property directly, which doesn't seem to be happening here. If you can reproduce this locally I can come help you debug.

**Comment by nickpell** [comment]
reproduction steps in the shell:

```
from corehq.apps.userreports.models import *
data_source_config = CustomDataSourceConfiguration.by_id('custom-airs-static-supervisory-indicators')
data_source_config.to_json()
```

**Pull request closed without merging**